nixos/attestation-server/module.nix

```diff
@@ -64,6 +94,22 @@ in
 
       serviceConfig = {
         ExecStart = "${cfg.package}/bin/AttestationServer";
+        ExecStartPre = let
+          values = concatStringsSep ", " [
+            "('emailUsername', '${cfg.emailUsername}')"
+            "('emailPassword', TRIM(readfile('%S/attestation/emailPassword'), char(10)))"
+            "('emailHost', '${cfg.emailHost}')"
+            "('emailPort', '${toString cfg.emailPort}')"
```

Comment:
I believe you should also set `emailLocal = 0` for the SELECT statement in `AlertDispatcher.java`.  When I tried this in the past, I didn't have any luck with `emailLocal = 1`, but I also didn't try very hard.

Reply:
I've added another option for that.